tests/wpt/mozilla/meta/css/css-position-3/position-sticky-grid.html.ini

```diff
@@ -0,0 +1,6 @@
+[position-sticky-grid.html]
+  type: reftest
+  expected: FAIL
+  [Untitled]
+    expected: FAIL
```

Comment:
Should we file bugs about the failing ones? Like, this one is pretty clear that it's because of missing grid support, but it'd be good to have issues on file for the inline ones for example.